demisto_sdk/__main__.py

```diff
@@ -3376,6 +3378,10 @@ def update_content_graph(
     is_flag=True,
     default=False,
 )
+@click.option(
+    "--mode",
```

Comment:
you can add enum to the mode to allow only specific values. See how we do that in `marketplace` option in other places